### PR TITLE
Copied more AMPHTML ad examples from amp.dev

### PR DESCRIPTION
**Pull request by lannka**

### CODE REVIEW

File: build-system/routes/a4a-envelopes.js [hunk 1]
```diff
@@ -35,6 +35,9 @@ app.use('/inabox/', (req, res) => {
     adUrl = urlPrefix.replace('localhost', 'ads.localhost') + adUrl;
   }
   adUrl = addQueryParam(adUrl, 'inabox', 1);
+  if (req.query.log) {
+    adUrl += '#log=' + req.query.log;
```
Comment (powerivq) [inline build-system/routes/a4a-envelopes.js]: Would this need url escaping?

File: examples/amphtml-ads/carousel-ad.a4a.html [hunk 2]
```diff
@@ -1,60 +1,202 @@
...
+    </div>
+    <div class="stretch">The All-New WX-S R5</div>
+    <a href="https://amp.dev/documentation/examples/" target="_blank" class="button">
+      Learn more
+    </a>
+  </div>
+</div>
 </body>
-</html>
+</html>
```
Comment (powerivq) [inline examples/amphtml-ads/carousel-ad.a4a.html]: Nit: keep the newline? Here and other .html

File: build-system/routes/a4a-envelopes.js (hunk 1 above)
Comment (lannka) [reply build-system/routes/a4a-envelopes.js]: should be fine, we only allow `log={number}`.

Comment (powerivq) [reply build-system/routes/a4a-envelopes.js]: K should be safe

**Pull request merged**